MN/Algoritmi/Algoritmi-ad-hoc/Deflatie.m

```diff
@@ -0,0 +1,32 @@
+% [USES] PutereDirecta(), PutereInversa()
+function [L, V] = Deflatie(A, max_iter, tol)
+  [n n] = size(A);
+  L = zeros(1, n);
+  V = zeros(n);
+  C = A;
+
+  for k = 1 : n - 1
+  % getting the current eigenvalue
+    [lam y nr] = PutereDirecta(A, max_iter, tol);
```

Comment:
- PowerMethod
- inverseaza ultimii doi param deoarece cel care lucreaza la sursa aceea i-a pus in alta ordine
- nu ai nevoie de nr, nu-l pune in vectorul de return